Sources/WasmInterpreter/WasmInterpreter+ImportHandler.swift

```diff
@@ -693,4 +693,134 @@ extension WasmInterpreter {
         )
         try self.importNativeFunction(named: name, namespace: namespace, signature: sig, handler: importedFunction)
     }
+    
+    public func addImportHandler<Arg1, Arg2, Arg3, Arg4, Arg5, Arg6, Arg7, Ret>(
```

Comment:
For consistency sake, it would be good to add the functions that don't include return values, as well. The offsets for these functions would start at 0, instead of at 1 for the functions that return values.

Reply:
I see where I missed this here, I'll get that implemented for the 7 and 8 parameter functions

Reply:
Resolved in latest commit